server/build/webpack.js

```diff
@@ -287,7 +282,7 @@ export default async function getBaseWebpackConfig (dir, {dev = false, isServer
           return count >= totalPages * 0.5
         }
       }),
-      !isServer && new webpack.optimize.CommonsChunkPlugin({
+      dev && !isServer && new webpack.optimize.CommonsChunkPlugin({
```

Comment:
This negates the react split bundle optimization with uglify. It might not be needed anymore because of the uglify optimizations we did though. cc @arunoda

Reply:
@ptomasroos let's remove this as well.

Reply:
Done